pisi/db/packagedb.py

```diff
@@ -114,12 +114,13 @@ def map_providers(doc, pkgConfigs: dict, pkgConfigs32: dict):
             return (pkgConfigs, pkgConfigs32)
 
         if repo is None:
-            for repo in repodb.list_repos():
+            repos = repodb.list_repos()
+            repos = repos[::-1] if repos is not None else None
```

Comment:
Could we use the Python `reverse()` function to make it more obvious what's going on here?

Reply:
didn't work on the repos object, can't remember the exact error

Reply:
I'm so sorry, I never saw a notification that you had replied >.<